src/mongocxx/collection.cpp

```diff
@@ -139,8 +138,8 @@ cursor collection::find(bsoncxx::document::view filter, const options::find& opt
         projection.bson(), rp_ptr));
 }
 
-stdx::optional<bsoncxx::document::value> collection::find_one(bsoncxx::document::view filter,
-                                                              const options::find& options) {
+stdx::optional<bsoncxx::document::value> collection::find_one(
+    bsoncxx::document::view_or_value filter, const options::find& options) {
```

Comment:
since this is a cpp can you just do

`using bsoncxx::document::view_or_value` at the top somewhere to get rid of the noise?